### ISSUE TITLE
update the midterm grading rubric

### ISSUE THREAD

**Issue by elliewix** [elliewix/IS-452-Fall2018#33]

**Comment by elliewix** [comment]
OK, this is done.

* text updated and new PDF done.
* I added more context and info about hard coding
* took out the essay info
* added text and the assignment to all the moodle assignments

This should be done now!

**Comment by elliewix** [comment]
Crap, wrong ticket.

**Comment by nicklalone** [comment]
Will get this done soon. Additionally, we can also start updating the solutions soon.

**Comment by elliewix** [comment]
Ankush put together the original, so he would be good to have help.

**Comment by ankush2712** [comment]
Is the midterm worth 70 points for 2 CR and 90 pts for 4 CR students?

**Comment by elliewix** [comment]
Yes, plus the 10 points for the check in that's due separately.  That might
be where the numbers are looking weird for you.

**Comment by ankush2712** [comment]
[Midterm Rubric Updated.xlsx](https://github.com/elliewix/IS-452-Fall2018/files/2349441/Midterm.Rubric.Updated.xlsx)
I see. So I leveraged the spring 2018 midterm rubric with the following modifications:
1. The total points add up to 70(2cr)/90(4cr) instead of 54 like spring 2018. So each individual steps weight (=percentage of total points) remains the same as in spring 2018 rubric.
2. I also included 10 pts for narrative. I somehow did not have any points for narrative and thought it should also be included like all the Homeworks.

Please let me know your thoughts on this.
